src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CaseDataCui.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CaseDataCui {
+
+    @JsonProperty("respondent1ResponseCui")
+    private RespondentResponseCui respondent1ResponseCui;
```

Comment:
Same applies here in LipResponse instead of CUI. In addition are we sure that in the lip case we have only to respondents and two applicants? if you look at cmc they have an array of respondents (defendants) and array of applicants (claimants) so potentially in cmc they can have more than two

Reply:
I think in XUI, you can only add 2. I remember seeing in CCD, all on respondent1 and respondent2 basis. The fields that are already in CCD, have that structure.

Reply:
Might be worth checking with the BAs as we will have multi-party coming in release 2 so probably better to add that ability sooner